frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -617,6 +634,21 @@ class TreesTabView extends React.PureComponent<Props, State> {
         <Menu.Item key="importNml" onClick={this.props.showDropzoneModal} title="Import NML files">
           <Icon type="upload" /> Import NML
         </Menu.Item>
+
+        <Menu.Item
+          key="measureSkeleton"
+          onClick={this.handleMeasureSkeletonLength}
+          title="Measure Skeleton Length"
+        >
+          <i className="fas fa-ruler" /> Measure Skeleton Length
+        </Menu.Item>
+        <Menu.Item
+          key="measureAllSkeletons"
+          onClick={this.handleMeasureAllSkeletonsLength}
+          title="Measure Length of all Skeletons"
```

Comment:
```suggestion
          title="Measure Length of All Skeletons"
```

According to https://capitalizemytitle.com/ because I was unsure as well :see_no_evil: